doc/introduction/photonic_hardware.rst

```diff
@@ -1,8 +1,8 @@
-Hardware
-========
+Xanadu Quantum Cloud
+====================
```

Comment:
This change makes sense, but at the same time I am hesitant - for a new user, seeing 'Hardware' in the side bar is much more descriptive and enticing than XQC.

Can we preserve 'hardware' somehow?

Reply:
Sure, "Hardware and cloud" might work? I cannot think of anything better really. 🤔